src/controllers/product.controller.js

```diff
@@ -0,0 +1,12 @@
+const { productService } = require("../services/product.service");
+
+const getProducts = async (req, res) => {
+  try {
+    res.status(200).json(await productService());
```

Comment:
```suggestion
const productService = require("../services/product.service");

const getProducts = async (req, res) => {
  try {
    res.status(200).json(await productService.getProducts());
```

다음과 같이 서비스 모듈을 통으로 불러와 활용할 수 있도록 하면 좋을 것 같아요~

Reply:
반영 완료했습니다.